tests/tests/test_demo_artifact.py

```diff
@@ -94,6 +95,7 @@ def run_demo_script_up(exit_cond=exit_cond):
 
     # Give the test a timeframe, as the script might run forever,
     # if something goes awry, or the script is not brought down properly.
+    @flaky(max_runs=3)  # https://tracker.mender.io/browse/MEN-4495
```

Comment:
This test started failing after traefik migration (iow post Mender 2.6).

I think we should not mark it `flaky` here

Reply:
I see, sure. Due to my somewhat stupid commit setup for this though, I think I'll actually add an extra commit in which I remove it from `test_demo` :-1: